tools/elastic/migration/Migrations/1.0.6/up/pre/00-pipeline-content.json

```diff
@@ -0,0 +1,33 @@
+{
+  "action": "createPipeline",
```

Comment:
Pipelines are a better solution than manually reindexing content, but for now we'll use both processes.